packages/components/tsconfig.esm.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
If we are to keep both tsconf, it would be better to have a base one and extend only with the necessary ones that differs between esm and cjs

Reply:
Yeah, can do that.

Reply:
But it's kind of pointless as compilerOptions doesn't get merged but overwritten so would have to duplicate all of them.

Reply:
So leaving as is.